resources/views/layouts/module.blade.php

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
```

Comment:
remove additional layout and use `app.blade.php` for dashboard view and `guest.blade.php` for un auth pages